tutorials/notebooks/imx500_notebooks/pytorch/pytorch_yolov8n_for_imx500.ipynb

```diff
@@ -343,58 +350,51 @@
     "                                         score_threshold=score_threshold,\n",
     "                                         iou_threshold=iou_threshold,\n",
     "                                         max_detections=max_detections).to(device=device)"
-   ],
-   "metadata": {
-    "tags": [
-     "long_run"
-    ],
-    "collapsed": false
-   },
-   "id": "240421e00f6cce34"
+   ]
   },
   {
    "cell_type": "markdown",
+   "id": "b5d72e8420550101",
+   "metadata": {
+    "collapsed": false
+   },
    "source": [
     "### Model Export\n",
     "\n",
     "Now, we can export the quantized model, ready for deployment, into a `.onnx` format file. Please ensure that the `save_model_path` has been set correctly. "
-   ],
-   "metadata": {
-    "collapsed": false
-   },
-   "id": "b5d72e8420550101"
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "546ff946af81702b",
+   "metadata": {
+    "collapsed": false,
+    "tags": [
+     "long_run"
+    ]
+   },
    "outputs": [],
    "source": [
     "mct.exporter.pytorch_export_model(model=gptq_quant_model_pp,\n",
```

Comment:
I think we better export the model without the "_pp", otherwise it can be confusing.. we need to emphasize that the "_pp" is for evaluation purpose.